daemon/daemon_linux.go

```diff
@@ -182,3 +183,22 @@ func ifaceAddrs(linkName string) (v4, v6 []*net.IPNet, err error) {
 	}
 	return v4, v6, nil
 }
+
+func (daemon *Daemon) supportsRecursivelyReadOnly(runtime string) error {
+	if kernel.CheckKernelVersion(5, 12, 0) {
```

Comment:
I’m not aware of any distro that backports this feature.

Reply:
Regardless of what has been backported so far, I'm strongly in favor of using a function check instead of a version number check; we've been bit by version number checks a lot in the past as I understand it.

Reply:
Updated to check the syscall